examples/chip-tool/commands/common/Commands.h

```diff
@@ -21,7 +21,7 @@
 #include "Command.h"
 #include <map>
 
-#include <controller/CHIPDeviceController.h>
+#include <controller/CHIPDeviceController_deprecated.h>
```

Comment:
the implication is that there's too much code to fix right now?

Reply:
Yes, there are about 4 apps that are using the old API. We can move those over to the new API one PR at a time, and delete deprecated API at that point.